wazo_calld/plugins/calls/services.py

```diff
@@ -309,6 +309,9 @@ def make_call_from_stasis_event(event):
         call.sip_call_id = channel_variables.get('WAZO_SIP_CALL_ID')
         call.line_id = channel_variables.get('WAZO_LINE_ID')
         call.creation_time = channel.get('creationtime')
+        call.is_caller = channel_variables.get('WAZO_CHANNEL_DIRECTION')
+        call.reason_code = event['cause']
```

Comment:
`reason_code` is not a part of `Call` object attribute. It should be added only when the event is created